backend/src/API/devPortfolioAPI.ts

```diff
@@ -3,36 +3,33 @@ import PermissionsManager from '../utils/permissionsManager';
 import { PermissionError, BadRequestError } from '../utils/errors';
 import { validateSubmission, isWithinDates } from '../utils/githubUtil';
 
-export const getAllDevPortfolios = async (
-  user: IdolMember,
-  isAdminRequest: boolean
-): Promise<DevPortfolio[]> => {
-  if (!isAdminRequest) {
-    return DevPortfolioDao.getAllInstances(false, user);
-  }
-
+export const getAllDevPortfolios = async (user: IdolMember): Promise<DevPortfolio[]> => {
   const isLeadOrAdmin = await PermissionsManager.isLeadOrAdmin(user);
   if (!isLeadOrAdmin)
     throw new PermissionError(
       `User with email ${user.email} does not have permission to view dev portfolios!`
     );
-  return DevPortfolioDao.getAllInstances(isLeadOrAdmin, user);
+  return DevPortfolioDao.getAllInstances();
 };
 
-export const getDevPortfolio = async (
+export const getAllDevPortfolioInfo = async (): Promise<DevPortfolioInfo[]> =>
+  DevPortfolioDao.getAllDevPortfolioInfo();
+
+export const getDevPortfolioInfo = async (uuid: string): Promise<DevPortfolioInfo> =>
+  DevPortfolioDao.getDevPortfolioInfo(uuid);
```

Comment:
Isn't this returning the whole dev portfolio object still? Since `DevPortfolioDao.getDevPortfolioInfo` just does this 
```
    const portfolioRef = await devPortfolioCollection.doc(uuid).get();
    return portfolioRef.data() as DevPortfolioInfo;
```

Reply:
Would you recommend using a delete operator? Or just setting the field to be undefined?

Reply:
Fixed.